Makefile

```diff
@@ -0,0 +1,27 @@
+IMAGE_NAME ?= whereabouts
+IMAGE_REGISTRY ?= ghcr.io/k8snetworkplumbingwg
+IMAGE_PULL_POLICY ?= Always
+IMAGE_TAG ?= latest
+
+OCI_BIN ?= docker
+
+build:
+	hack/build-go.sh
+
+docker-build:
+	$(OCI_BIN) build -t ${IMAGE_REGISTRY}/${IMAGE_NAME}:${IMAGE_TAG} -f Dockerfile .
+
+generate-api:
+	hack/verify-codegen.sh
+	rm -rf github.com
+
+install-tools:
+	hack/install-kubebuilder-tools.sh
+
+test: build install-tools
+	hack/test-go.sh
+
+test/e2e:
+	hack/e2e-get-test-tools.sh
+	hack/e2e-setup-kind-cluster.sh
+	$(GO) test github.com/k8snetworkplumbingwg/whereabouts/whereabouts_e2e
```

Comment:
I need to require the `KUBECONFIG` env variable somehow for this target.

Reply:
Probably no need... Let's just error, and request the user to set it. This is the thrown error - fairly easy to see what you have to do ...

```
• Failure in Spec Setup (BeforeEach) [0.000 seconds]
Whereabouts functionality
/home/mduarted/github/whereabouts/e2e/e2e_test.go:43
  Test setup
  /home/mduarted/github/whereabouts/e2e/e2e_test.go:44
    stateful set tests [BeforeEach]
    /home/mduarted/github/whereabouts/e2e/e2e_test.go:194
      regular sized network
      /home/mduarted/github/whereabouts/e2e/e2e_test.go:206
        stateful sets scale up / down
        /home/mduarted/github/whereabouts/e2e/e2e_test.go:243
          scale down then up 20 replicas
          /home/mduarted/github/whereabouts/e2e/e2e_test.go:279

          Unexpected error:
              <*errors.errorString | 0xc00031d700>: {
                  s: "must provide the path to the kubeconfig via the `KUBECONFIG` env variable",
              }
              must provide the path to the kubeconfig via the `KUBECONFIG` env variable
          occurred

          /home/mduarted/github/whereabouts/e2e/e2e_test.go:71

```